reference/perf-benchmarks.md

```diff
@@ -1,17 +1,34 @@
-The following benchmarks give an overview of the performance that you can achieve with Anbox Cloud. The numbers were collected on various virtual and bare metal hardware.
+The following benchmarks give an overview of the performance that you can achieve with Anbox Cloud.
+
+The benchmarks are performed using the `amc benchmark` utility as described [here](https://discourse.ubuntu.com/t/benchmarking-a-deployment/17770). The results describe the maximum number of parallel running containers (colum "# Containers") delivering a stable frame rate (column "Avg. FPS"). Running more containers either gives to high variation in the provided frame rate or is not possible due to other hardware limitions (system memory, GPU memory, ...).
+
+All benchmark include rendering and video encoding. On machines/VMs without a GPU rendering and video encoding are performed in software on the CPU.
+
+For most of the benchmarks below a special version of the application [BombSquad](https://www.froemling.net/apps/bombsquad) is used. It runs in a demo mode where the game provides random and automated simulated gameplay. This allows the benchmark to simulate a real world scenario where actual users would play the game instead of sitting on a static game scene without much variation across the Android instances.
```

Comment:
```suggestion
For most of the benchmarks below, a special version of the [BombSquad](https://www.froemling.net/apps/bombsquad) application was used. This version runs in a demo mode in which the game provides random and automated simulated gameplay. This allows the benchmark to simulate a real-world scenario where actual users would play the game, instead of sitting on a static game scene without much variation across the Android instances.
```